app/lib/receipt.rb

```diff
@@ -0,0 +1,28 @@
+module Receipt
+
+  def print_receipt_head(shop_details)
+    receipt_header = [time_and_date]
+    shop_details.each { |detail| receipt_header << detail }
+    receipt_header
+  end
+
+  def print_receipt_body(order)
+    receipt_body = []
+    order.each { |item, details| receipt_body << "#{item} x #{details[0]} = $#{details[1].round(2)}" }
+    receipt_body
+  end
+
+  def print_receipt_footer(order_list, adjustments, cash)
+    receipt_footer = ["#{adjustments.item_on_discount} Discount 10% from $#{adjustments.item_discount_total(order_list).round(2)}", 
```

Comment:
Useless assignment to variable - `receipt_footer`.<br>Line is too long. [132/80]<br>Trailing whitespace detected.